migrations/tests_without_build_definition.py

```diff
@@ -30,7 +30,7 @@ def is_package_config_affected(package_config: str) -> bool:
         build_jobs_with_same_trigger = [
             job
             for job in jobs
-            if job.get("job") in (JobType.copr_build.value, JobType.build.value)
+            if job.get("job") == JobType.copr_build.value
```

Comment:
Are these changes relevant though?

Reply:
This were the "old" changes, we can no more import `JobType.build.value` from `packit` once we release the new version. This should fix the code for the new packit release.